panic: json: cannot unmarshal number 10881715073803520453 into Go struct field DomainAPISimplifiedEvaluationLogicItemV1.resources.apps.evaluation_logic.simplified_logic.id of type int64

Making the following request (pulled from the debug output of gofalcon):

```
GET /spotlight/combined/vulnerabilities/v1?facet=cve&facet=host_info&facet=remediation&facet=evaluation_logic&filter=status%3A%27open%27 HTTP/1.1
```

I get the following error:

```
panic: json: cannot unmarshal number 10881715073803520453 into Go struct field DomainAPISimplifiedEvaluationLogicItemV1.resources.apps.evaluation_logic.simplified_logic.id of type int64

goroutine 19 [running]:
main.queryVulnerabilities.func1()
	/home/klape/src/acs/crowdstrike-integration/main.go:162 +0x519
created by main.queryVulnerabilities in goroutine 1
	/home/klape/src/acs/crowdstrike-integration/main.go:148 +0xcb
```

The max size of an `int64` is `9223372036854775807`. Comparing the two numbers:

```
9223372036854775807
10881715073803520453
```

The number appears twice in the payload, once in the above field, and another one that seems to parse: `DomainAPIEvaluationLogicItemV1.resources.apps.evaluation_logic.logic.id`.

* [Working type code definition (`jsonext.Number`)](https://github.com/CrowdStrike/gofalcon/blob/f368c62485a99fd718976949a38a45e0cfa86883/falcon/models/domain_api_evaluation_logic_item_v1.go#L39)
* [Broken type code definition (`int64`)](https://github.com/CrowdStrike/gofalcon/blob/f368c62485a99fd718976949a38a45e0cfa86883/falcon/models/domain_api_simplified_evaluation_logic_item_v1.go#L33)

--- a/falcon/models/domain_api_simplified_evaluation_logic_item_v1.go
+++ b/falcon/models/domain_api_simplified_evaluation_logic_item_v1.go
@@ -7,6 +7,7 @@ package models
 
 import (
 	"context"
+	jsonext "encoding/json"
 
 	"github.com/go-openapi/errors"
 	"github.com/go-openapi/strfmt"
@@ -30,7 +31,7 @@ type DomainAPISimplifiedEvaluationLogicItemV1 struct {
 	Found []string `json:"found"`
 
 	// id
-	ID int64 `json:"id,omitempty"`
+	ID jsonext.Number `json:"id,omitempty"`
 
 	// match required
 	MatchRequired string `json:"match_required,omitempty"`

--- a/specs/transformation.jq
+++ b/specs/transformation.jq
@@ -5,8 +5,9 @@
   | .paths."/intel/entities/rules-latest-files/v1"."get"."responses"."200"."schema"={"$ref": "#/definitions/domain.DownloadItem"}
   | .paths."/intel/entities/rules-files/v1"."get"."responses"."200"."schema"={"$ref": "#/definitions/domain.DownloadItem"}
   | .paths."/real-time-response/entities/extracted-file-contents/v1"."get"."responses"."200"."schema"={"$ref": "#/definitions/domain.DownloadItem"}
-  # Fix overflow on json number (more than 63 bits are neede hold this field)
+  # Fix overflow on json number (more than 63 bits are needed hold this field)
   | .definitions."domain.APIEvaluationLogicItemV1".properties.id."x-go-type"={type: "Number", import: {package: "encoding/json"}, hints: {noValidation: true}}
+  | .definitions."domain.APISimplifiedEvaluationLogicItemV1".properties.id."x-go-type"={type: "Number", import: {package: "encoding/json"}, hints: {noValidation: true}}
   # Rename msaspec.Error back to msa.APIError. These are two names for the same type.
   | walk(
     if type == "object" and has("$ref") and ."$ref" == "#/definitions/msaspec.Error" then ."$ref" = "#/definitions/msa.APIError" else . end

